packer/plugin.go

```diff
@@ -47,6 +48,8 @@ type PluginConfig struct {
 // without being confused with spaces in the path to the command itself.
 const PACKERSPACE = "-PACKERSPACE-"
 
+const OS_ARCH = runtime.GOOS + "_" + runtime.GOARCH
```

Comment:
Since this const is not used anywhere else in the code base I would suggest unexporting it prevent unexpected reuse. 
```suggestion
const osArch  = runtime.GOOS + "_" + runtime.GOARCH
```

Reply:
Sounds good, I renamed this in the newest commit